winui/Ribbon/RibbonQAT.md

```diff
@@ -0,0 +1,185 @@
+          </ribbon:QuickAccessToolBar.MenuItems>
+      </ribbon:QuickAccessToolBar>        
+   </ribbon:SfRibbon.QuickAccessToolBar>
+   ...
+</ribbon:SfRibbon>
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI Ribbon QAT menu items](Ribbon-QAT-images/winui-ribbon-qat-menu-items.png)
+
+N> To avoid adding duplicate items in QAT, set Content property for ribbon items.
+
+## Removing QAT items
+
+To remove an items from Quick Access Toolbar (QAT), right-click the required QAT item and select `Remove from Quick Access Toolbar` option. 
+
+![WinUI Ribbon QAT item context menu](Ribbon-QAT-images/winui-ribbon-qat-item-context-menu.png)
+
+![WinUI Ribbon QAT item removed](Ribbon-QAT-images/winui-ribbon-qat-item-removed.png)
+
+## Change Position of Quick Access Toolbar
+
+The position of Quick Access Toolbar (QAT) can be changed by using it `Position` property.
+
+The Position enumeration comprises of following values: .
+
+* **Above the Ribbon** - Displays the Quick Access Toolbar above the ribbon. 
+* **Below the Ribbon** - Displays the Quick Access Toolbar below the ribbon.
+* **Hide** - Hide the Quick Access Toolbar in the ribbon.
+
+The default value is **Above the Ribbon**. The below code shows how to change the position of `Quick Access Toolbar` in Ribbon.
+
+{% tabs %}
+{% highlight xaml %}
+<ribbon:SfRibbon x:Name="ribbon">
+   <ribbon:SfRibbon.QuickAccessToolBar>
+      <ribbon:QuickAccessToolBar Position="BelowRibbon">
+          <ribbon:RibbonButton x:Name="SaveButton"
+                               Content="Save"
+                               Icon="Save"
+                               Command="{Binding ButtonCommand}"
+                               CommandParameter="Save"/>
+          <ribbon:RibbonButton x:Name="undoButton"
+                               Content="Undo"
+                               Icon="Undo"
+                               Command="{Binding ButtonCommand}"
+                               CommandParameter="Undo"/>
+          <ribbon:RibbonButton x:Name="printButton"
+                               Content="Print"
+                               Icon="Print"
+                               Command="{Binding ButtonCommand}"
+                               CommandParameter="Print"/>                       
+       </ribbon:QuickAccessToolBar>
+    </ribbon:SfRibbon.QuickAccessToolBar>
+    ...
+</ribbon:SfRibbon>
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI Ribbon QAT below ribbon](Ribbon-QAT-images/winui-ribbon-qat-below-ribbon.png)
+
+The position of the QAT can also be changed using its drop-down menu option.
+
+![WinUI Ribbon QAT drop-down menu](Ribbon-QAT-images/winui-ribbon-qat-drop-down-menu.png)
+
+## Command Label
+
+WinUI Ribbon Command Label illustrates the implementation of MS Word. When the QAT is positioned below the Ribbon, it supports command labels at runtime. To show or hide the command label for QAT items, click `Show Command Labels` or `Hide Command Labels` in the Context menu

```

Comment:
You can show or hide the command label in the Quick Access Toolbar using the `Show Command Labels` or `Hide Command Labels` QAT drop-down menu item.

N> `Show Command Labels` or `Hide Command Labels` menu item will only be shown in QAT drop-down while placing QAT below the ribbon like in Microsoft Word.